examples/tensor/fixed_rank_tensor.cpp

```diff
@@ -74,7 +74,17 @@ int main() {
     
     std::cout<<"( exp2 == exp3 ) => "<<( exp2 == exp3 ? "True" : "False")<<'\n';
 
+    /*--------------------------Assignment------------------------------*/
 
+    // Assigning static_tensor into fixed_rank_tensor
+    ub::fixed_rank_tensor<float,5> t4 = st1;
+
+    // Assigning dynamic_tensor into fixed_rank_tensor
+    ub::dynamic_tensor t5 = dt1;
+
+    std::cout<<"( t4 == t3 ) => "<<( t4 == t3 ? "True" : "False")<<'\n';
+    
+    std::cout<<"( t5 == dt1 ) => "<<( t5 == dt1 ? "True" : "False")<<'\n';
 
```

Comment:
I would use `std::endl` and `std::boolalpha` and rewrite
```cpp
std::cout<< "(t4== t3) => "<< std::boolalpha << (t4==t3) << std::endl;
```
Its more readable and there is really big difference in the assembler output:
See [godbolt](https://godbolt.org/z/N3xEQ4)

Reply:
I have changed it. I was using `'\n'` because `std::endl` does two tasks the first one is new line and the second one is flush and I did not want to use flush and in case of `std::boolalpha` it will change of the `std::cout` and if you want to revert back you again have to set the state. I think `std::format` will be a better replacement for formating than `std::cout`.

Reply:
It am nitpicking here. I would prefer `std::boolalpha`. This
```cpp
    std::cout << std::boolalpha;
    std::cout << "(t4==t3)" << (t4==t3) << std::endl;
    std::cout << "(t5==t2)" << (t5==t2) << std::endl;
```
easier to read. We can still rewrite it at a later time using `std::format`.